src/stores/linecolors.spec.js

```diff
@@ -0,0 +1,30 @@
+import assert from 'assert';
+import {LobsterStore} from '../stores';
+
+test('Store-Line-Colors', function() {
+  const data = [
+    '[ShardedClusterFixture:job0:configsvr:secondary0]',
```

Comment:
Can you replace SharedClusterFixture with something different in every line? Anything at all works! Also can you increment job0 on a couple lines.